src/index.js

```diff
@@ -0,0 +1,189 @@
+  }
+};
+
+const resetCity = (event) => {
+  const resetTheCity = document.querySelector('#cityNameDisplay');
+  resetTheCity.textContent = 'Seattle';
+  const city = document.querySelector('#cityName');
+  city.value = '';
+};
+
+const changeLandscape = (event) => {
+  if (state.temp <= 59) {
+    document.querySelector('#weatherGarden').textContent =
+      '🌲🌲⛄️🌲⛄️🍂🌲🍁🌲🌲⛄️🍂🌲';
+  } else if (state.temp > 59 && state.temp <= 69) {
+    document.querySelector('#weatherGarden').textContent =
+      '🌾🌾_🍃_🪨__🛤_🌾🌾🌾_🍃';
+  } else if (state.temp > 69 && state.temp <= 79) {
+    document.querySelector('#weatherGarden').textContent =
+      '🌸🌿🌼__🌷🌻🌿_☘️🌱_🌻🌷';
+  } else if (state.temp > 79) {
+    document.querySelector('#weatherGarden').textContent =
+      '🌵__🐍_🦂_🌵🌵__🐍_🏜_🦂';
+  }
+};
+const inputCity = (event) => {
+  const city = document.querySelector('#cityName');
+  const displayName = document.querySelector('#cityNameDisplay');
+  displayName.textContent = city.value;
+};
+
+const changeSky = (event) => {
+  const skySelector = document.querySelector('#skySelector');
+  const sky = document.querySelector('#skyDisplay');
+  if (skySelector.value === 'cloudy') {
+    document.querySelector('#skyDisplay').textContent =
+      '☁️☁️ ☁️ ☁️☁️ ☁️ 🌤 ☁️ ☁️☁️';
+  } else if (skySelector.value === 'raining') {
+    document.querySelector('#skyDisplay').textContent = '🌧🌈⛈🌧🌧💧⛈🌧🌦🌧💧🌧🌧';
+  } else if (skySelector.value === 'snowing') {
+    document.querySelector('#skyDisplay').textContent = '🌨❄️🌨🌨❄️❄️🌨❄️🌨❄️❄️🌨🌨';
+  } else if (skySelector.value === 'sunny') {
+    document.querySelector('#skyDisplay').textContent = '☁️ ☁️ ☁️ ☀️ ☁️ ☁️';
+  }
+};
+
+const loadCityTemp = (event) => {
+  const city = document.querySelector('#cityName');
+  const searchString = city.value;
+  const tempPromise = getTempFromSearch(searchString);
+  const tempCount = document.querySelector('#temperatureDisplay');
+  tempPromise.then((temp) => {
+    state.temp = temp;
+    tempCount.textContent = ` ${state.temp} ℉`;
+    changeTempColor();
+  });
+};
+
+const registerEventHandlers = (event) => {
+  const increaseTemp = document.querySelector('#increaseTemperature');
+  increaseTemp.addEventListener('click', increaseTemperature);
+
+  const decreaseTemp = document.querySelector('#reduceTemperature');
+  decreaseTemp.addEventListener('click', decreaseTemperature);
+
+  const changeColorOnHotter = document.querySelector('#increaseTemperature');
+  changeColorOnHotter.addEventListener('click', changeTempColor);
+
+  const changeColorOnColder = document.querySelector('#reduceTemperature');
+  changeColorOnColder.addEventListener('click', changeTempColor);
+
+  const defaultCity = document.querySelector('#cityReset');
+  defaultCity.addEventListener('click', resetCity);
+
+
+  const newLandscapeOnDecreacse = document.querySelector('#reduceTemperature');
+  newLandscapeOnDecreacse.addEventListener('click', changeLandscape);
```

Comment:
To handle all the things that should happen when the temperature changes, I prefer putting all the actions together into a single handler each for the increase and descrease buttons. It makes it more explicitly clear that all those things happen together. Any time the temperature increases or decreases, regardless of how that happens, I waant the color and landscape to also update. Currently, the lansdscape doesn't update as the temperature increases, since there's no registration for `changeLandscape` on `increaseTemperature `.